### PR TITLE
Clarify TechOps, GovPress, and D+ relationship

### PR DESCRIPTION
**Pull request by thill-dxw**
Moving Technical Operations to its own top-level section; added clarifying text to Development index.md

### CODE REVIEW

**Comment by thill-dxw** [comment]
As discussed a couple of weeks ago with @snim2 and @rjw1, this doesn't do much more than clarify that there are in fact three related technology-related teams at dxw. [Other changes](https://github.com/dxw/playbook/pull/971#issuecomment-1236846996) may be desirable, but can be dealt with separately.

**Review by yndajas — APPROVED** [review]
Looks good - one very small thing in the comment~~s~~

File: src/tech/index.md [hunk 1]
```diff
@@ -5,6 +5,10 @@ related_order: 1000
 
 ## Development
 
+At dxw, developers typically work in multidisciplinary teams that include delivery managers, interaction designers, and other specialists to design and develop high-quality bespoke applications to meet user needs.
+
+In addition, dxw contains two other technology-related job families with a narrower focus. To learn more about these, please see the Playbook pages dedicated to [Technical Operations](/technical-operations) and to the [GovPress business unit](/govpress-unit/); this section is focused on the work of developers dedicated to custom work as part of a multidisciplinary group.
```
[the remark this answers is not in the document]
Comment (thill-dxw) [reply src/tech/index.md]: Fixed!

**Comment by snim2** [comment]
@thill-dxw This looks good, but looking through the current Playbook, there are quite a few tech-ops related bits and pieces, and I'm wondering whether you intend to move those into the new directory? (and if so, we need to add some redirects to the old pages!)

**Comment by thill-dxw** [comment]
@snim2 - I was thinking that reorganising the tech-ops content would be a job best undertaken by @rjw1: this PR just gives him an obvious directory for doing that. I'm happy to do that editing myself - but then feel that @rjw1 would just end up having to review it closely anyway.

**Comment by rjw1** [comment]
I do think the tech page should just be saying there are 3 parts to it and development gets split out to its own page as well. Just so we don't continue the lie that tech is only development.

**Comment by rjw1** [comment]
lots of the bits are thigns the entire company needs to know and arent about the tech ops team.

**Comment by thill-dxw** [comment]
@rjw1 Reviewing this more comprehensively (and having taken a look on the recommendation of @james at his [Bikeshed post](https://bikeshed.dxw.com/2022/10/17/what-technical-operations-does/) on Tech Ops), the difficulty is pointed to by your observation that

> lots of the bits are things the entire company needs to know and aren't about the tech ops team.

On the one hand Tech Ops is distinctive and has its own domain; on the other, it's also embedded in Delivery+, Commercial Operations, and GovPress. Treating Delivery+, Tech Ops, and GovPress as three independent pillars thus doesn't quite make sense.

I think what's needed to reflect the reality of the situation is:

* me to revise and expand the text of the Technology landing page to better articulate the points of overlap with Technical Operations
* somebody (@snim2?) to do the same wrt GovPress
* somebody (presumably @rjw1) to populate the new top-level landing page for Technical Operations with a description of Technical Operations and its many roles, and linking out to all the various references to these currently scattered throughout the Playbook

Does that make sense?

**Comment by snim2** [comment]
> @rjw1 Reviewing this more comprehensively (and having taken a look on the recommendation of @james at his [Bikeshed post](https://bikeshed.dxw.com/2022/10/17/what-technical-operations-does/) on Tech Ops), the difficulty is pointed to by your observation that
> 
> > lots of the bits are things the entire company needs to know and aren't about the tech ops team.
> 
> On the one hand Tech Ops is distinctive and has its own domain; on the other, it's also embedded in Delivery+, Commercial Operations, and GovPress. Treating Delivery+, Tech Ops, and GovPress as three independent pillars thus doesn't quite make sense.
> 
> ...

I'm not sure.

Like @rjw1 says, some things apply to all of us, and some apply to only one of the three teams (D+/GovPress/Tech Ops). Some things (e.g. OOH support) are led by one team, but involve all three.

I think probably what we need to do first is to figure out which parts of the playbook apply to which group(s), and then once we've got a page new structure figured out, the three of us can write any missing parts of the docs for our group. 

If we start by re-writing that first page, I wonder whether we'll just end up re-writing it several more times as we go along?

**Comment by jdudley1123** [comment]
Do we want to merge this or ditch it?

**Pull request closed without merging**